unit/util/sharing_map.cpp

```diff
@@ -492,7 +492,7 @@ TEST_CASE("Sharing map views and iteration", "[core][util]")
 
   SECTION("View")
   {
-    typedef std::pair<std::string, std::string> pt;
+    typedef std::pair<irep_idt, std::string> pt;
```

Comment:
Suggest changing this back to `std::string`. The intention was to sort the vector in lexicographic order. However, `<` for `irep_idt` sorts according to their sequence number AFAIK.